docs/deployment/customization/portal.properties-Reference.md

```diff
@@ -218,6 +218,19 @@ skin.patient_view.copy_number_table.columns.show_on_init=
 skin.patient_view.structural_variant_table.columns.show_on_init=
 ```
 
+### Define custom sample type colors
+Define the colors of custom sample types in the patient view using a json object with for each sample type a color:
+```
+skin.patient_view.custom_sample_type_colors_json=classpath:/skin-patient-view-custom-sample-type-colors.json
```

Comment:
@inodb this documentation is wrong now, correct?  i mean, why is there a classpath there?